test/integration/model.test.js

```diff
@@ -2198,7 +2198,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
 
         if (dialectName === 'postgres') {
           test++;
-          expect(table.id.defaultValue).to.not.contain('special');
+          expect(table.id.defaultValue).to.contain('special');
```

Comment:
`describeTable` should use the default schema if the schema is unspecified. This change doesn't reflect that as the result comes from the "special" (non default) schema

Reply:
Fixed